test/build.info

```diff
@@ -28,7 +28,7 @@ INCLUDE_MAIN___test_libtestutil_OLB = /INCLUDE=MAIN
           aborttest test_test \
           sanitytest exdatatest bntest \
           ectest ecstresstest ecdsatest gmdifftest pbelutest ideatest \
-          sm2sigtest sm2crypttest \
+          sm2_sign_internal_test sm2_crypt_internal_test \
```

Comment:
Since you're adding these in `PROGRAMS_NO_INST` later on, you can (should!) remove them here.

Reply:
@levitte is this the only change requested prior to merge? Thanks!